### PR TITLE
Improve Zookeeper connections management

### PR DESCRIPTION
**Pull request by rvassallo-stratio**
Improve Zookeeper connection management assuring Zookeeper connection is connected and alive before returning it.
Additionally, some minor changes have been done in order to remove warnings during compilation.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ hose {
     RELEASETIMEOUT = 30
     FOSS = true
     REPOSITORY = 'common-utils'
+    DEPLOYONPRS = true
```
Comment (jmboyero-stratio) [inline Jenkinsfile]: This should be setted to false on final version.

```suggestion
    DEPLOYONPRS = false
```

Comment (rvassallo-stratio) [reply Jenkinsfile]: Yes, it is just for testing purposes.
It will be removed before merging.

File: pom.xml [hunk 2]
```diff
@@ -115,6 +115,12 @@
             <artifactId>slf4j-api</artifactId>
             <version>${slf4j.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>1.7.16</version>
```
Comment (mafernandez-stratio) [inline pom.xml]: Maybe a property for the version?

Comment (rvassallo-stratio) [reply pom.xml]: Fixed :+1:

File: Jenkinsfile (hunk 1 above)
Comment (rvassallo-stratio) [reply Jenkinsfile]: Line removed :+1:

**Pull request merged**